settings.py

```diff
@@ -331,7 +331,6 @@
     'custom.world_vision',
     'custom.tdh',
     'custom.up_nrhm',
-    'crispy_forms',
```

Comment:
Was removing crispy_forms intentional here? Just wanted to make sure because it seems unrelated to the rest of the PR